Makefile

```diff
@@ -40,16 +40,12 @@ ifneq ($(T1),)
 $(T1):
 	@echo >/dev/null
 else
-	include unity/Makefile
-	include stdio/Makefile
-	include proc/Makefile
-	include mem/Makefile
-	include fs/Makefile
-	include disk/Makefile
-	#include virtio/Makefile
-	include sample/Makefile
-	include setjmp/Makefile
 
+ifneq ($(TARGET_SUFF),host)
```

Comment:
maybe instead of making special conditionals here we should add `include Makefile.common` to `Makefile.arm` (and others)?